tasks/configure.js

```diff
@@ -26,9 +26,16 @@ function fetchFromNextConfigVars(source, target, key) {
 
 function fetchFromVault(source, target) {
 	console.log(`Fetching ${source} config from the vault for ${target}`);
-	const pathPrefix = 'secret/teams/next';
-	return vault.get()
-		.then(vault => vault.read(`${pathPrefix}/${source}/production`))
+
+	const normalizedSource = normalizeName(source);
+
+	const path = fetch('https://next-registry.ft.com/v2/')
+		.then(fetchres)
+		.then(json => json.find(app => app.name === normalizedSource).config)
```

Comment:
I'd get rid of `const normalizedSource = normalizeName(source);` line and modify this line to become:

`.then(json => json.find(app => app.name === normalizeName(source)).config)`